### ISSUE TITLE
error 

### ISSUE THREAD

**Issue by daredevil88** [VictorNJ21/esx_blackmarket#1]
![errorblackmarket](https://user-images.githubusercontent.com/20977555/89928753-9708b200-dc08-11ea-99f3-a5af12f4275e.jpg)

getting error about canCarryItem

**Comment by VictorNJ21** [comment]
Do you know which ESX version you are using? Because this script is made for one of the newer ESX versions, if you're using an old one, it won't work.

**Comment by daredevil88** [comment]
> Do you know which ESX version you are using? Because this script is made for one of the newer ESX versions, if you're using an old one, it won't work.

yes latest

**Comment by VictorNJ21** [comment]
ESX 2? Or ESX V1 Latest?

**Comment by daredevil88** [comment]
as i see esx2 is still work in progress and not stable do you have a version i can use

**Comment by VictorNJ21** [comment]
Correct, ESX 2 is WIP and unstable, I recommend using es_extended V.1.1.0 converted to weight, or es_extended V.1.2.0/V1 latest. You can find all the releases here: https://github.com/esx-framework/es_extended/releases
